recipe/meta.yaml

```diff
@@ -10,33 +10,22 @@ source:
 
 build:
   noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv && {{ PYTHON }} -c 'import pysr; pysr.install()'
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
-  build:
-    - python
-    - pip
-    - pyjulia >=0.5.7
   host:
     - python >=3.6,<4.0
     - pip
-    - pyjulia >=0.5.7
-    - julia <=1.7.1
-    - numpy >=1.14.0
-    - pandas
-    - sympy
-    - scikit-learn
-    - openlibm <0.8.0
   run:
     - python >=3.6,<4.0
     - pyjulia >=0.5.7
     - julia <=1.7.1
     - numpy >=1.14.0
     - pandas
     - sympy
-    - scikit-learn
-    - openlibm
+    - scikit-learn >=1.0.0
+    - openlibm <0.8.0
```

Comment:
PySR itself works for any Julia version above 1.5.

I saw issues with the conda-forge version of julia 1.7.2 when I tried it a while ago, and since most PySR users don't use Julia for other things I thought it would be fine to fix it at a stable version. I could try relaxing that requirement now.

Reply:
I think we can defend against this by checking whether [`CONDA_BUILD_STATE`](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#dynamic-behavior-based-on-state-of-build-process) is not `test` in the `post-link` script.